version/generate_version

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# generates the version of file based on current git version
+
+cd $GOPATH/src/github.com/contiv/netplugin/version
+
+USE_RELEASE=$1
+
+if [ -z "${USE_RELEASE}" ]
+then
+  file_name=version_default.go
+  UseVersionStr="// +build !use_version" 
```

Comment:
@jainvipin , may be this comment got overlooked. Can you kindly address how you plan to use `use_version` build label and if it needs to go in this patch?